src/Firebase/firebase-event-interface.ts

```diff
@@ -14,6 +22,7 @@ export interface PuzzleCompletedEvent extends CommonEventProperties {
     target: string;
     foils: string[];
     response_time: number;
+    event_date_with_timestamp: string;
```

Comment:
move event_date_with_timestamp to commonEventProperties, because this propeties is in all events